src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy

```diff
@@ -147,11 +153,13 @@ class HockeyAppUploadTask extends DefaultTask {
             }
         }
 
-        uploadAppplicationFileToHockeyApp(applicationFile, mappingFile, appId)
+        applicationFiles.each {
+            uploadApplicationFileToHockeyApp(it, mappingFile, appId)
```

Comment:
For us it's not a problem: we're going to upload splitted APKs for internal use only (and use direct links, not auto-update) and we still get crash reports from every APK, grouped by build number in one place. But yes, looks like there is no full support of the splitted APKs and maybe will not in the near future.